### PR TITLE
Adjust for OpenFunction v0.8.1-rc.0 & support kubeversion v1.25

### PR DESCRIPTION
**Pull request by wrongerror**
Fixes: 
- https://github.com/OpenFunction/OpenFunction/issues/386 
- https://github.com/OpenFunction/OpenFunction/issues/376
- https://github.com/OpenFunction/OpenFunction/issues/362
- https://github.com/OpenFunction/OpenFunction/issues/379

Signed-off-by: wrongerror <wangyifei@kubesphere.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # openfunction
 
-![Version: 0.3.1](https://img.shields.io/badge/Version-0.3.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0](https://img.shields.io/badge/AppVersion-0.8.0-informational?style=flat-square)
+![Version: 0.3.2-rc.0](https://img.shields.io/badge/Version-0.3.2--rc.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.1-rc.0](https://img.shields.io/badge/AppVersion-0.8.1--rc.0-informational?style=flat-square)
```
Comment (benjaminhuo) [inline README.md:3]: Maybe we can use `0.4.0` directly instead of `0.3.2-rc.0`.

Comment (wrongerror) [reply README.md:3]: The rc version is required ? corresponding to OpenFunction v0.8.0-rc.0

Comment (benjaminhuo) [reply README.md:3]: ok then maybe 0.4.0-rc.0 ?

Comment (wrongerror) [reply README.md:3]: ok

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**